legacy/src/main/java/ca/bc/gov/app/repository/ForestClientRepository.java

```diff
@@ -98,6 +98,13 @@ Flux<ForestClientEntity> findClientByIncorporationOrName(
           OR c.client_identification = :value
           OR UTL_MATCH.JARO_WINKLER_SIMILARITY(c.legal_middle_name,:value) >= 90
           OR c.legal_middle_name LIKE '%' || :value || '%'
+          OR (c.client_type_code = 'I' AND (
```

Comment:
Please add the exact condition to the countByPredictiveSearch method

Reply:
Done. sorry; it was an oversight on my part.